UefiCpuPkg/PiSmmCpuDxeSmm/Ia32/SmmProfileArch.c

```diff
@@ -12,13 +12,17 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 /**
   Create SMM page table for S3 path.
 
+  @param[out] Cr3    The base address of the page tables.
+
 **/
 VOID
 InitSmmS3Cr3 (
-  VOID
+  OUT UINT64  *Cr3
   )

```

Comment:
CR3 is 32bit in 32bit env. Using UINT64 is strange.

Reply:
ok, i will change to UINTN